app/Repositories/Eloquents/BookmetaEloquentRepository.php

```diff
@@ -97,4 +97,26 @@ public function destroyCountReview($id)
             return $e->getMessage();
         }
     }
+
+    public function updateReturned($id,$data)
```

Comment:
**Reporter**: CHECKSTYLE</br>**Rule**: Squiz.Functions.FunctionDeclarationArgumentSpacing.NoSpaceBeforeArg</br>**Severity**: error</br>**File**: app/Repositories/Eloquents/BookmetaEloquentRepository.php</br>**Line**: 101</br>Expected 1 space between comma and argument "$data"; 0 found